core/unit_test/CMakeLists.txt

```diff
@@ -65,10 +66,10 @@ macro(Cabana_add_tests)
           if(_device STREQUAL Pthread OR _device STREQUAL OpenMP)
             foreach(_thread ${CABANA_UNIT_TEST_NUMTHREADS})
               add_test(NAME ${_target}_${_thread} COMMAND
-                ${_target} ${gtest_args} --kokkos-threads=${_thread})
+                ${NONMPI_PRECOMMAND} ${_target} ${gtest_args} --kokkos-threads=${_thread})
```

Comment:
@junghans admittedly it is probably not the right way to set the number of threads with `jsrun` but this will do for now

Reply:
ok